x-pack/plugins/lens/public/drag_drop/drag_drop.scss

```diff
@@ -0,0 +1,7 @@
+.drag-drop_is-drop-target {
```

Comment:
Can you rename these classes to use BEM naming structure? (Guidelines at the bottom of [this page](https://elastic.github.io/eui/#/guidelines/sass)).

Also, you'll need a 3-letter prefix for scoping.

Reply:
Will do. I haven't done much CSS since joining Kibana. I hadn't seen those guidelines before. I've historically not used camelCasing in CSS, but it looks like that's the norm here, is that right?

Reply:
That's correct. Those guidelines should explain how to name your classes. There's also a link to what BEM naming is.